apps/financial-aid/backend/src/environments/environment.ts

```diff
@@ -53,7 +53,7 @@ const devConfig = {
   },
   identityServerAuth: {
     issuer: 'https://identity-server.dev01.devland.is',
-    audience: '@samband.is',
+    audience: '@island.is', // TODO: Figure out how we support both island.is and samband audience
```

Comment:
@eirikurn Is there someone using IDS currently which allows 2 audiences?

In our case we would like to allow '@island.is' and '@samband.is' and then we would do extra ACL in our backend.

Reply:
Why do you want to have a `@island.is` audience? We should try and keep your backend audience/scopes completely separate from @island.is.

Reply:
Because when this goes into the application system, we will use the same login as the application system does for the applicant flow, which means the audience in the user jwt will be @island.is. 

Is there another way of doing this that I am missing?

Reply:
Yeah, I think you might have this backwards. APIs are "Resource Servers" in OAuth2 lingo. We define these resources in IDS, for example named "@island.is" and "@samband.is" and associate scopes with them.

When the Application System client gets access to some scopes (eg "@island.is/applications:read" and "@samband.is/financial-aid:read"), the IDS issues a JWT with audience = all the associated resource servers.

It means the audience for this token are resource servers which handle the list of scopes. Then when I send it to different APIs, they verify that the token was "meant" for that API by checking if the audience matches.

tldr: tokens can have multiple audiences. If a token has a samband scope, it will have a samband audience (resource) in the token as well. Your API should only check that.

Reply:
Makes sense, thanks for the write up :)